src/styles/blocks/header.scss

```diff
@@ -0,0 +1,60 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 40px 0 50px;
+
+  .logo-container a {
+    display: block;
+
+    .logo-img {
+      display: block;
+    }
+  }
+
+  .nav__list {
```

Comment:
nav is a separate BEM-block, should be in a separate file
write this element inside block property:
```
.nav {
  &__list {
   // some properties
  }
}
```